model/model.go

```diff
@@ -926,7 +933,8 @@ func zeroOf(t Type, pm map[string]string, pkgOverride string) string {
 			if pkgOverride == t.Package || t.Package == "" {
 				return t.Name + "{}"
 			} else {
-				return t.Package + "." + t.Name + "{}"
+				// TODO: need the package name, not the package path.
+				return filepath.Base(t.Package) + "." + t.Name + "{}"
```

Comment:
Seems to me like we should just use the package's imported name in the `Package` field instead of trying to figure it out here.

Reply:
`t.Package` is a path. Where can I get the name of the package?

Reply:
Is `pm[t.Package]` correct?

Reply:
I meant that we should store the name in the `Package` field instead.

Reply:
I haven't looked into how the generation is done. Will most likely not look at it at all this round.

Reply:
`pm[t.Package]` is fine; my point here was that we could store that value in the struct instead, on line 704 of this file.

Either way, could you remove the TODO now?

Reply:
> my point here was that we could store that value in the struct instead, on line 704 of this file.

Another PR.

Reply:
This is fine as-is.

Reply:
Cool.